main.py

```diff
@@ -63,19 +63,18 @@ def add_command(self, cmd: list, hlp: str, fn, access_level: AccessLevel, params
     def get_single_arg(self, context: CallbackContext) -> str:
         return self.hubitat.case_hack(" ".join(context.args))
 
-    def get_device(self, update: Update, context: CallbackContext) -> Device:
+    def get_devices(self, update: Update, context: CallbackContext) -> list[Device]:
         device_name = self.get_single_arg(context)
         if not device_name:
             self.send_text(update, context, "Device name not specified.")
-            return None
+            return list()
 
-        device = self.hubitat.resolve_device(device_name, self.get_user(update).device_groups)
+        devices = self.hubitat.resolve_devices(device_name, self.get_user(update).device_groups)
 
-        if device:
-            return device
+        if not devices:
+            self.send_text(update, context, "Device not found. '/l' to get list of devices.")
```

Comment:
This can a little bit more helpful. Resolve_devices knows what device is not found, it can help the human.